caesars-palace/blocks/header/header.js

```diff
@@ -129,6 +129,62 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+const showMore = (nav) => {
+  const ul = nav.querySelector('.nav-sections > .local-nav > ul');
+  const more = document.createElement('li');
+  more.classList.add('more');
+  const moreText = document.createElement('div');
+  moreText.classList.add('more-text');
+  const aMore = document.createElement('a');
+  aMore.classList.add('more-link');
+  aMore.text = 'MORE';
+  const downArrow = document.createElement('div');
+  downArrow.classList.add('down-arrow');
+  aMore.appendChild(downArrow);
```

Comment:
You may not need to add a `div` for the down-arrow. It could potentially be added using CSS only via an `::after` pseudo element.

Reply:
I think we need to style the arrow and add event listeners to it... Which is probably why it's in its own div?